Cargo.toml

```diff
@@ -24,6 +24,7 @@ keywords = [
 features = ["nightly", "numpy"]
 
 [dependencies]
+no-std-compat = { version = "0.4.1", features = [ "alloc", "compat_hash" ] }
```

Comment:
@antimora Is no_std still useful even if alloc will be required?

Reply:
@coreylowman Awesome! Thank you for adding the support. Yes, it still will be very useful. Sorry, I missed your message earlier. I came back to see the status of this request and you added it! Great, thanks!